### ISSUE TITLE
`media-feature-range-operator-space-after` passes over an operator standing behind another

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#240]
Split out of [#226](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/226), which carries three bugs that share nothing but the review that found them. This is the third of the three, and its account there is not quite right: the rule does converge, in two runs rather than one, and what is wrong is the reading rather than the writing.

### What minimal example or steps are needed to reproduce the bug?

A set of parameters holding one range operator directly behind another:

```css
@media ,a<>=b screen<screen { a { b: c; } }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/media-feature-range-operator-space-after": "always"
  }
}
```

`media-feature-range-operator-space-before` reads through the same utility and goes the same way.

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

Every range operator given the whitespace the option asks for, in one run.

### What actually happened?

```
input   @media ,a<>=b screen<screen { a { b: c; } }
run 1   @media ,a< >=b screen< screen { a { b: c; } }
run 2   @media ,a< >= b screen< screen { a { b: c; } }
run 3   unchanged
```

The `>=` is not seen on the first run at all. It is seen on the second, once a space has been written in front of it. A file holding a longer run of operators needs a run of the fixer for each of them, and a user who runs `--fix` once is left with warnings the fixer says it fixed.

### What is the cause?

`findMediaOperator` guards against reporting the `=` of `>=` as an operator of its own by looking at the character standing in front of a match:

```js
let before = params[match.startIndex - 1]

if (before === `>` || before === `<`) return
```

That also passes over a whole `>=` wherever a `<` or a `>` stands in front of it.

### Do you have a proposal to fix the bug?

Read the operator, then decide, rather than deciding from the character in front: skip a match that begins inside the one before it, by the end index of the previous match.

The shape `@media ,a<>=b screen<screen` is not a query any browser reads, and its fixed form is not meaningful either. That is fine — the rule's job is the whitespace, and a shape it cannot make sense of must still come out the same on the second run as on the first.
